### PR TITLE
Embed YouTube videos of cohort calls

### PR DESCRIPTION
**Pull request by NPDebs**
This PR fixes issue open-life-science/open-life-science.github.io#592.
I added an `<iframe>` tag where the video url is extracted.
Then, I put the proper embed code in the YouTube links.

Embed_issue

**Note for Reviewers**
The video for Week 5 (Cohort 2) cannot play on external sites.

EmbedPR

### CODE REVIEW

**Comment by NPDebs** [comment]
Hi, @AndreaSanchezTapia & @bebatut. 👋
Please review this when you get the chance. Thank you! ❤️

File: _includes/week.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## {{ c.type }} call: {{ c.title }}
 
 {% if c.type != "Mentor-Mentee" and c.date %}
-<i class="fas fa-calendar-alt"></i> **Date**: {{ c.date }}{% if c.time %}, at [{{ c.time }} Universal Time](https://arewemeetingyet.com/UTC/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/
```
Comment (bebatut) [inline _includes/week.md:6]: Can you revert all your changes on the time? Thanks

File: _includes/week.md [hunk 2]
```diff
@@ -28,7 +28,8 @@
 
 **Resources**:
 - <i class="fas fa-clipboard"></i> [Notes]({{ c.notes }})
-- <i class="fab fa-youtube"></i> {% if c.recording %} [Recording]({{ c.recording }}) {% else %} Recording available on the [OLS YouTube channel]({{ site.youtube }}) after the call {% endif %}
+- <i class="fab fa-youtube"></i> {% if c.recording %} Resources
```
Comment (bebatut) [inline _includes/week.md:31]:
```suggestion
- <i class="fab fa-youtube"></i> {% if c.recording %} Recording
```

File: _includes/week.md [hunk 3]
```diff
@@ -28,7 +28,8 @@
...
+<iframe src="{{ c.recording }}"></iframe> {% else %} Recording available on the [OLS YouTube channel]({{ site.youtube }}) after the call {% endif %}
```
Comment (bebatut) [inline _includes/week.md:32]: It would be nice to make the video bigger in the page there

Comment (bebatut) [reply _includes/week.md:32]: Maybe an idea: having on the side of the call detail

File: _includes/week.md (hunk 1 above)
Comment (NPDebs) [reply _includes/week.md:6]: Oh wow. I only just noticed the time changes now. Will revert immediately, please...

Comment (NPDebs) [reply _includes/week.md:6]: Done. 👍

File: _includes/week.md (hunk 3 above)
Comment (NPDebs) [reply _includes/week.md:32]: 💭 That's a  good suggestion. I was concerned that having a bigger video might give us responsiveness issues.

I'll try to do a sample soon and get your thoughts on it. Thanks!

File: _data/ols-3-schedule.yaml [hunk 4]
```diff
@@ -218,7 +218,7 @@ weeks:
       duration: 30 min
       title: Open office hour, coworking on assignments, knowledge exchange and networking
       date: February 24, 2021
-      time: '17:00'
+      time: '18:00'
```
Comment (bebatut) [inline _data/ols-3-schedule.yaml:221]:
```suggestion
      time: '17:00'
```

File: _data/ols-4-schedule.yaml [hunk 5]
```diff
@@ -299,11 +299,11 @@ weeks:
       notes: https://bit.ly/ols-4-week-07
       title: Q&A call
       type: Q&A
-  08:
+  '08':
     start: November 01, 2021
     calls:
     - date: November 03, 2021
-      time: '17:00'
+      time: '16:00'
```
Comment (bebatut) [inline _data/ols-4-schedule.yaml:306]:
```suggestion
      time: '17:00'
```

File: _data/ols-4-schedule.yaml [hunk 6]
```diff
@@ -331,14 +331,14 @@ weeks:
...
-  09:
+  '09':
     start: November 08, 2021
     calls:
     - duration: 30 min
       title: 5th mentor-mentee meeting
       type: Mentor-Mentee
     - date: November 10, 2021
-      time: '12:30'
+      time: '11:30'
```
Comment (bebatut) [inline _data/ols-4-schedule.yaml:341]:
```suggestion
      time: '12:30'
```

File: _data/ols-5-schedule.yaml [hunk 7]
```diff
@@ -5,7 +5,7 @@ timeline:
   description: Call for Application opens on [Open Review](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5)
   details: See the [guidelines and templates](https://github.com/open-life-science/application-forms)
 - date: December 14, 2021
-  time: '16:00'
+  time: '17:00'
```
Comment (bebatut) [inline _data/ols-5-schedule.yaml:8]:
```suggestion
  time: '16:00'
```

File: _data/ols-5-schedule.yaml [hunk 8]
```diff
@@ -41,7 +41,7 @@ weeks:
       title: Mentor onboarding group 1
       type: Mentor
     - date: February 22, 2022
-      time: '09:00'
+      time: '10:00'
```
Comment (bebatut) [inline _data/ols-5-schedule.yaml:44]:
```suggestion
      time: '09:00'
```

File: _includes/week.md (hunk 1 above)
Comment (bebatut) [reply _includes/week.md:6]: What do you mean?

File: _data/ols-2-schedule.yaml [hunk 9]
```diff
@@ -700,10 +700,10 @@ weeks:
         duration: 90 min
         title: 'Open Leadership: Academia, industry and beyond!'
         date: November 19, 2020
-        time: '17:30'
+        time: '15:30'
```
Comment (bebatut) [inline _data/ols-2-schedule.yaml:703]:
```suggestion
        time: '17:30'
```

**Comment by bebatut** [comment]
Implemented in #693

**Pull request closed without merging**